README.md

```diff
@@ -77,3 +77,19 @@ Here is a TL;DR:
 
 And finally, as the person merging all PRs other reviewers or I (Gk/erasels) may ask you to make changes to your Pack in case some cards lie outside the power curve or skirt the rules too much.
 Please don't be discouraged if your Pack isn't instantly merged.
+
+### HATS
+
+Packmaster can equip different cosmetic hats, one for every pack. Each one is locked until you've won a run with its respective pack in your pool.
+
+This does not add any mandatory work for packs. There are three ways to get a hat added to your pack:
+
+1) Art it yourself, using the new Hat template PSD.  Add a png export of your hat into the resources/hats folder, with the naming convention of <PackID>Hat.png, and it will automatically be detected and used.
```

Comment:
Let's point people towards the specific file name to reduce the chance of confusion.

```suggestion
1) Art it yourself, using the new Hat template PSD (PackmasterHatTemplate.psd, included in this repo).  Add a png export of your hat into the resources/hats folder, with the naming convention of <PackID>Hat.png, and it will automatically be detected and used.
```